tests/test_modeling_gpt_neo.py

```diff
@@ -238,6 +239,17 @@ def create_and_check_lm_head_model(self, config, input_ids, input_mask, head_mas
         self.parent.assertEqual(result.loss.shape, ())
         self.parent.assertEqual(result.logits.shape, (self.batch_size, self.seq_length, self.vocab_size))
 
+    def create_and_check_gpt_neo_for_sequence_classification(
+        self, config, input_ids, input_mask, head_mask, token_type_ids, mc_token_ids, sequence_labels, *args
+    ):
```

Comment:
we need to add a test for this in `GPTNeoModelTest` class, otherwise, it won't run.
see the GPT2 test https://github.com/huggingface/transformers/blob/afe479adb5474250215438fe27db9dc9dbbbde09/tests/test_modeling_gpt2.py#L460-L463

Reply:
Thanks for the suggestion!
I will fix it.

Reply:
Thanks for the suggestion!
I will fix it.